content/blog/newsletter_2022-02-05.md

```diff
@@ -0,0 +1,236 @@
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+## Events
+
+[XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Videos
+
+Thilo Molitor (developer of Monal) [held a talk](https://xmpp-meetup.in-berlin.de/talks/monal.mp4) (DE) about Monal's development.
+
+XMPP Office Hours: Fabian Sauter presented his [adventures in developing an XMPP Client](https://www.youtube.com/watch?v=Bceu6E7c7GM) for Windows (Universal Windows Platform (UWP)) in December.
+
+XMPP has been mentioned in a [German public TV](https://youtu.be/DdIwus87eLM?t=844) [DE] show in the context of data protection.
+
+## Articles
+
+JMP.chat released two blog posts. [The first](https://blog.jmp.chat/b/2022-jabber-xmpp-from-sms) details a feature of the Soprani.ca project's Cheogram system that allows SMS users to contact (or call!) any XMPP address. [Their Newsletter](https://blog.jmp.chat/b/january-newsletter-2022) also announces a partnership with Snikket for hosting, as well as a preview of worldwide calling rates as they prepare to launch that feature soon.
+
+There are several articles to the topic  "messenger" at the German site from "[Freie Messenger](https://www.freie-messenger.de/#neuigkeiten-letzte-änderungen)" with focus on alternatives to WhatsApp, E2EE, interoperability, security/pseudosecurity. You may help translating them to your native language.
+
+[OMEMO was finally integrated in Movim](https://mov.im/?post/pubsub.movim.eu/Movim/end-to-end-encryption-in-movim-omemo-is-finally-there-yudZPP) after 6 long years of discussions. In this article we are explaining what is the general OMEMO architecture what were the difficulties encountered when doing the integration in Movim and how we overcame them.
+
+As the [previously announced](https://snikket.org/blog/simply-secure-collaboration/) collaboration between Snikket and Simply Secure ended its first project, they interviewed the project's founder, Matthew Wild, about Snikket's origins and his experience managing open-source projects. Read the interview: [On Getting Things Done: A Conversation with Matthew Wild from Snikket](https://simplysecure.org/blog/on-getting-things-done-a-conversation-with-matthew-wild-from-snikket/).
+
+Mellium Co-op has published their [Year in Review for 2021](https://opencollective.com/mellium/updates/mellium-year-in-review-2021) and the [Dev Communiqué for December 2021](https://opencollective.com/mellium/updates/dev-communique-for-december-2021).
```

Comment:
"and for [January 2022](https://opencollective.com/mellium/updates/dev-communique-for-january-2022)]" could be added at the end, sorry for the confusion.

Reply:
f41ce92397b1e9ed037405a84d0f35234edc1d83